Documentation/Presentations.md

```diff
@@ -0,0 +1,224 @@
+
+![Modal size - Half](https://raw.githubusercontent.com/IBAnimatable/IBAnimatable-Misc/master/IBAnimatable/PresentationFullHalf.png)
+
+### Full
+
+| Value | Effect |
+| ------------- | ------------- |
+| `Full` | Use the full size of the screen (width or height) |
+
+*The following screenshot is using `Full` as with, and `Full` as height*
+
+![Modal size - Full](https://raw.githubusercontent.com/IBAnimatable/IBAnimatable-Misc/master/IBAnimatable/PresentationFull.png)
+
+### Custom
+
+| Value | Effect |
+| ------------- | ------------- |
+| `Custom(Float)` | Set a custom value |
+
+*The following screenshot is using `Custom(300)` as with, and `Custom(300)` as height*
+
+![Modal size - Custom](https://raw.githubusercontent.com/IBAnimatable/IBAnimatable-Misc/master/IBAnimatable/PresentationSizeCustom.png)
+
+## Presentation Animators
+
+`IBAnimatable` provide a broad set of Presentation Animators. We can use them in Interface Builder as **Presentation Animation** and **Dismiss Animation** as described in [Configuring a Presentation in Interface Builder](#configuring-a-presentation-in-interface-builder), or programmatically in code. They are all standard Transition Animators conform to `UIViewControllerAnimatedTransitioning`.
+
+You can see all supported Transition Animators in the demo App, open the App and tap on "Playground" button, then tap on "presentations", then play with the different options.
+
+### Flip
+
+| Value | Effect |
+| ------------- | ------------- |
+| `Flip ` | |
+
+![Transition - Fade Transition](https://raw.githubusercontent.com/IBAnimatable/IBAnimatable-Misc/master/IBAnimatable/PresentationFlip.gif)
+
+### CrossDissolve
+
+| Value | Effect |
+| ------------- | ------------- |
+| `CrossDissolve` | Cross fade |
+
+![Transition - Fade Transition](https://raw.githubusercontent.com/IBAnimatable/IBAnimatable-Misc/master/IBAnimatable/CrossdissolveAnimator.gif)
+
+### Cover
+
+| Value | Effect |
+| ------------- | ------------- |
+| `Cover(Top)` | Slide in / out from top |
+| `Cover(Right)` | Slide in / out from right |
+| `Cover(Bottom)` | Slide in / out from bottom |
+| `Cover(Left)` | Slide in / out from left |
+
+![Transition - Fade Transition](https://raw.githubusercontent.com/IBAnimatable/IBAnimatable-Misc/master/IBAnimatable/CoverAnimator.gif)
+
+### Zoom
+
+| Value | Effect |
+| ------------- | ------------- |
+| `Zoom` | Zoom effect |
+
+![Transition - Fade Transition](https://raw.githubusercontent.com/IBAnimatable/IBAnimatable-Misc/master/IBAnimatable/ZoomAnimator.gif)
+
+### DropDown
+
+| Value | Effect |
+| ------------- | ------------- |
+| `DropDown` | Slide in then slide out with a drop effect |
+
+![Transition - Fade Transition](https://raw.githubusercontent.com/IBAnimatable/IBAnimatable-Misc/master/IBAnimatable/DropDownAnimator.gif)
+
+## Keyboard Translations
+
+If your modal contains an `UITextField` / `UITextView`, you can adapt its origin when the keyboard is opening
```

Comment:
If your modal contains a (an is not right here sine U pronounce /juː/) `UITextField` or `UITextView`, you can adjust (not sure here, I think adapt is ok too) its position (position may be clearer for designers, origin is UIKit specific term) when the keyboard is opening.